pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/GoogleAuth.scala

```diff
@@ -99,4 +93,40 @@ class GoogleAuth(config: GoogleAuthSettings, system: String, redirectUrl: String
       }
     }
   }
-}
\ No newline at end of file
+
+
+  private def requestToken(tokenEndpoint: String, code: Seq[String], retries: Int = 3): Future[WSResponse] = {
+    WS.url(tokenEndpoint).post {
+      Map(
+        "code" -> code,
```

Comment:
I don't know, but if so the other error will just break the recursion and be passed to the caller?